File: tys-spa/ciudadano/src/app/domain/correspondencia-tramite.ts
import {Remitente} from './remitente';
import {DireccionTerritorial} from './direccion-territorial';

export interface CorrespondenciaTramite {
  idTipoComunicacion?: number;
  idMedioRecepcion?: number;
  idTipologiaDocumental?: number;
  tiempoRespuesta?: number;
  fechaRadicacion?: string;
  numeroRadicado?: string;
  asunto?: string;
  descripcion?: string;
  referidos?: Array<string>;
  remitente?: Remitente;
  listaDireccionTerritoriales?: Array<DireccionTerritorial>;
  idUsuario?: string;
}

File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/CorrespondenciaTramite.java
package co.com.mintrabajo.tys.commons.domain;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class CorrespondenciaTramite {

	private Long idTipoComunicacion;
	private Long idMedioRecepcion;
	private Long idTipologiaDocumental;
	private Long tiempoRespuesta;
	private String fechaRadicacion;
	private String numeroRadicado;
	private String asunto;
	private String descripcion;
	private List<String> referidos;
	private Remitente remitente;
	private List<DireccionTerritorial> listaDireccionTerritoriales;
	private String idUsuario;
	private String nombreDependencia;
	
	public CorrespondenciaTramite(){
		
	}

}
